README.md

```diff
@@ -164,9 +164,9 @@ Supported Platforms
 
 - iOS
 - Android 6+
-- Windows Phone 7
-- Windows Phone 8
 - Windows Phone 8.1
+- Windows 8.1
+- Windows 10
```

Comment:
Does it actually do anything on Windows 10?

Reply:
Hm, I don't think so:
https://github.com/apache/cordova-plugin-statusbar/blob/003fa610307f0d2b68aed2cebdc04a939d77912f/src/windows/StatusBarProxy.js#L25-L32

`Windows.UI.ViewManagement.StatusBar` is probably
https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.statusbar

. | .
-- | --
Device family | Windows Mobile Extension SDK (introduced v10.0.10240.0)
API contract | Windows.Phone.PhoneContract (introduced v1)

This probably means that it is only present on Phones?

Reply:
Care to test? You seems the Windows expert :)

Reply:
Problem is I have no way to test the Win8.1 thing to check if it works at all right now. Seems I have to install VS15 again. Fun.

Reply:
To unblock this PR: My proposal is to remove the Windows 8.1 and 10 lines to keep the README as it is. Fine with you?